thunder/clang/__init__.py

```diff
@@ -280,6 +280,13 @@ def full_like(
     return full(a.shape, fill_value, device=device, dtype=dtype)
 
 
+@clangop()
+def empty(shape: Sequence[int], *, device: DeviceLike, dtype: dtypes.dtype) -> TensorLike:
+    device = devices.to_device(device)
+
+    return prims.empty(shape, device=device, dtype=dtype)
```

Comment:
`tuple(shape)` since `shape` can be a `Sequence[int]` here

Reply:
Ah, I missed that. Thank you for catching!